ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleConsent.java

```diff
@@ -32,4 +32,21 @@ public static void updateConsentStatus(Vungle.Consent consentStatus,
   public static String getCurrentVungleConsentMessageVersion() {
     return sCurrentVungleConsentMessageVersion;
   }
+
+  /**
+   * Update CCPA consent status.
+   * @param status See {@link Vungle.Consent}.
+   *               If true, the user has consented to us gathering data about their device.
+   */
+  public static void setCCPAStatus(Vungle.Consent status) {
+    Vungle.updateCCPAStatus(status);
```

Comment:
@HuFangshuai @palrahul could we do the same for updateConsentStatus? We don't really need a static sCurrentVungleConsent & sCurrentVungleConsentMessageVersion because VugleSDK does the same...